lib/scarpe/para.rb

```diff
@@ -1,13 +1,25 @@
 class Scarpe
   class Para
-    def initialize(app, text)
+    def initialize(app, text, stroke: nil, **html_attributes)
       @app = app
       @text = Array(text)
+      @stroke = stroke
+      @html_attributes = Hash(html_attributes)
```

Comment:
🤔 I think the double splat ensures that `html_attributes` will always be a hash even if none are passed so this should be able to be just:
```suggestion
      @html_attributes = html_attributes
```